org.adempiere.base/src/org/compiere/model/MDepositBatchLine.java

```diff
@@ -197,4 +220,9 @@ private void updateHeader()
 		DB.executeUpdateEx(sql, new Object[] {getC_DepositBatch_ID()}, get_TrxName());
 	}	//	updateHeader
 	
+	@Override
+	public MPayment getC_Payment() throws RuntimeException {
+		return new MPayment(getCtx(), getC_Payment_ID(), get_TrxName());

```

Comment:
should be `return getC_Payment_ID() > 0 ? new MPayment(getCtx(), getC_Payment_ID(), get_TrxName()) : null;` instead.

Reply:
fixed

Reply:
Fixed